src/common/OperationsBase.ts

```diff
@@ -3,7 +3,7 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import isomorphicFetch from "cross-fetch";
-import { DataType, ECProperty } from "./interfaces/GroupProperties";
+import { DataType, ECProperty } from "../reporting/interfaces/GroupProperties";
```

Comment:
Nothing under `common` should be importing anything from `reporting` or `carbon-calculation`.

Reply:
moved isValid function to mappingsClient since it was the only client using it.